### PR TITLE
Improve Ubuntu build instructions 

### LINKED ISSUE #1770: Improve Ubuntu build instructions
**Issue summary**

In the readme it is written:
```
# Ubuntu
sudo apt install build-essential clang ocl-icd-opencl-dev libssl-dev
```

This is not enough for building forest on Ubuntu 20.04.4 LTS.
It looks like `m4` and `pkg-config` are missing.

And I think we should always be explicit about which version of Ubuntu we are speaking.

**Other information and links**

### PR DESCRIPTION
**Pull request by tyshko5**
**Summary of changes**
Changes introduced in this pull request:
- 

**Reference issue to close (if applicable)**

Closes #1770

**Other information and links**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,10 +65,10 @@ rustup install nightly
 * OpenCL bindings
 
 ```shell
-# Ubuntu
+# Ubuntu (20.04)
 sudo make install-deps
 # Or
-sudo apt install build-essential clang ocl-icd-opencl-dev libssl-dev
+sudo apt-get install --no-install-recommends -y build-essential clang ocl-icd-opencl-dev aria2 cmake
```
Comment (lemmih) [inline README.md:71]: Let's stick to `apt`. And `aria2` is an optional dependency.

**Review by lemmih — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tyshko5) [reply README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -65,10 +65,10 @@ rustup install nightly
 * OpenCL bindings
 
 ```shell
-# Ubuntu
+# Ubuntu (20.04)
 sudo make install-deps
 # Or
-sudo apt install build-essential clang ocl-icd-opencl-dev libssl-dev
+sudo apt install --no-install-recommends -y build-essential clang ocl-icd-opencl-dev cmake
```
Comment (lemmih) [inline README.md:71]: This doesn't belong here: `-no-install-recommends -y`

**Review by lemmih — CHANGES_REQUESTED** [review]

**Comment by elmattic** [comment]
Would be great to have per OS command line, where you just can copy the command line and paste it, like in lotus:

lotus_cmdline

File: README.md (hunk 2 above)
Comment (tyshko5) [reply README.md]: Fixed

**Review by hanabi1224 — APPROVED** [review]

**Comment by lemmih** [comment]
@tyshko5 You can apply @elmattic's suggestion in this PR or we can worry about it later. When you're satisfied, feel free to merge this PR.

**Pull request merged**